10up/wp-local-docker-v2
# Add: PHP 8.0

### Description of the Change
This PR adds PHP 8.0 support using `10up/wp-php-fpm-dev:8.0` image.


### Verification Process
1. Uninstall `wp-local-docker` and install this branch using `npm install https://github.com/dinhtungdu/wp-local-docker-v2\#cf27c20b -g`.
2. Run `10updocker create`
3. See PHP 8.0 option.


### Checklist:






### Applicable Issues
Fixes #190


### Changelog Entry


Added support for PHP 8.0.
